Updated log levels in pkg

akremsa opened the pull request:
This PR can be considered as a part of #1223 

Updated log levels in pkg package.

aLekSer commented on pkg/sdkserver/localsdk.go:
```diff
@@ -125,7 +125,7 @@ func NewLocalSDKServer(filePath string) (*LocalSDKServer, error) {
 	go func() {
 		for value := range l.update {
-			logrus.Info("gameserver update received")
+			logrus.Debug("gameserver update received")
```
Nit: Not connected to the logLevel, but for consistency with log messages above.
```suggestion
			logrus.Debug("GameServer update received")
```

markmandel commented on CONTRIBUTING.md:
```diff
@@ -101,3 +101,9 @@ repository, we have a [community membership guide](./docs/governance/community_m
 
 - [How to write a good Git Commit message](https://chris.beams.io/posts/git-commit/) -
   Great way to make sure your Pull Requests get accepted.
+- **Log levels usage:**
```
Please move this entire section it's own PR. Where possible, smaller PRs are better, especially PRs that touch on multiple aspects.

Here we have a PR that is about log levels in SDK Server, but also touches on contributing guidelines. It's very hard to approve / reject both.

akremsa commented:
@markmandel  thanks for the review. I've removed changes from the md file and added other log levels updates to this PR because all of them are related to the same package - pkg, and I think it's better to keep everything in a single PR.

akremsa commented:
/assign @markmandel

aLekSer commented:
@markmandel I think we could merge this PR before the release. It would significantly reduce logging in SDK server with Info level used by default.
